docs/supported-libraries.md

```diff
@@ -17,111 +17,128 @@ or [contributing](../CONTRIBUTING.md).
+| [Kubernetes Client](https://github.com/kubernetes-client/java)                                                                    | 7.0+                          | TODO                                                                                   |
+| [Lettuce](https://github.com/lettuce-io/lettuce-core)                                                                             | 4.0+                          | TODO                                                                                   |
+| [Log4j 1](https://logging.apache.org/log4j/1.2/)                                                                                  | 1.2+                          | TODO                                                                                   |
+| [Log4j 2](https://logging.apache.org/log4j/2.x/)                                                                                  | 2.11+                         | TODO                                                                                   |
+| [Logback](http://logback.qos.ch/)                                                                                                 | 1.0+                          | TODO                                                                                   |
+| [Micrometer](https://micrometer.io/)                                                                                              | 1.5+                          | TODO                                                                                   |
+| [MongoDB Driver](https://mongodb.github.io/mongo-java-driver/)                                                                    | 3.1+                          | TODO                                                                                   |
+| [Netty](https://github.com/netty/netty)                                                                                           | 3.8+                          | [HTTP Server Spans], [HTTP Client Spans], [HTTP Server Metrics], [HTTP Client Metrics] |
+| [OkHttp](https://github.com/square/okhttp/)                                                                                       | 2.2+                          | TODO                                                                                   |
+| [Oracle UCP](https://docs.oracle.com/database/121/JJUCP/)                                                                         | 11.2+                         | TODO                                                                                   |
+| [OSHI](https://github.com/oshi/oshi/)                                                                                             | 5.3.1+                        | [System Metrics]                                                                       |
```

Comment:
maybe something about it being a partial implementation of system metrics, maybe even list the metrics that it emits?

Reply:
Enumerating the differences seems like the responsibility of the readme in the oshi module. Probably good enough to indicate that its a partial implementation IMO.